discovery/sync_manager_test.go

```diff
@@ -537,10 +538,14 @@ func assertTransitionToChansSynced(t *testing.T, s *GossipSyncer, peer *mockPeer
 	}
 	assertMsgSent(t, peer, query)
 
-	s.ProcessQueryMsg(&lnwire.ReplyChannelRange{
+	require.Eventually(t, func() bool {
```

Comment:
woah, didn't know they had this

Reply:
https://www.youtube.com/watch?v=GHe8kKO8uds